app/backend/src/tests/test_visible_users.py

```diff
@@ -0,0 +1,24 @@
+import pytest
+
+from couchers.db import get_user_by_field
+from couchers.models import User
+from tests.test_fixtures import db, generate_user, make_user_invisible, session_scope
+
+
+def test_visible_user_filter(db):
+    user1, token1 = generate_user()
+    user2, token2 = generate_user()
```

Comment:
Just this should work:

```py3
user2, token2 = generate_user(is_banned=True)
```

Reply:
It doesn't. Auth interceptor gets it. Try it yourself

Reply:
Ah... sorry about that.

How about `session.query(User).filter(User.id == user2.id).one().is_banned = True`? Then no need to refresh/expunge the user since it's not written to

Reply:
It works without the expunge and refresh, just forgot to take them out.

Reply:
Would you prefer a query instead of get_user_by_field?

Reply:
Yes please